.github/workflows/node-sdk.yml

```diff
@@ -0,0 +1,67 @@
+name: Whispir Node SDK generator
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  Create-Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +"%Y-%m-%dT%H-%M-%S%z")"
+      - name: date
+        run: echo ${{steps.date.outputs.date}}
+      # Checkout whispir/openapi repo
+      - uses: actions/checkout@v2
+      # Clone whispir/whispir-node
+      - name: Clone Whispir-Node
+        if: success()
+        uses: whispir/checkout@main
+        with:
+          repository: whispir/${{secrets.GH_REPO}}
+          ref: "main"
+          path: whispir-node
+          token: ${{ secrets.GH_TOKEN }}
+      # Create release branch
+      - name: Create release branch
+        run: |
+          cd whispir-node/
+          git checkout -b release/${{steps.date.outputs.date}}
```

Comment:
What would happen if we have multiple releases on the same date? We may need some degree of random uniqueness for these release branches, or alternatively a more precise date